rmgpy/test_data/testing_database/kinetics/libraries/lib_net/reactions.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+name = "lib_net"
+shortDesc = u""
+longDesc = u"""
+This is a fake library for testing PDep networks exploration of library reactions
+"""
+
+entry(
+    index = 1,
+    label = "CH3 + CH3 <=> ethane",
+    degeneracy = 1.0,
+    kinetics = Chebyshev(
+        coeffs = [
+            [12.8791, 0.560009, -0.115177, 0.0105093],
+            [-1.04561, 0.857712, -0.116582, -0.0134747],
+            [-0.599769, 0.449793, 0.0096248, -0.0271013],
+            [-0.314421, 0.177204, 0.0458462, -0.0107188],
+            [-0.155503, 0.0517415, 0.0323201, 0.00343069],
+            [-0.0727158, 0.00622718, 0.0138549, 0.00630263],
+        ],
+        kunits = 'cm^3/(mol*s)',
+        Tmin = (300, 'K'),
+        Tmax = (3000, 'K'),
+        Pmin = (0.001, 'bar'),
+        Pmax = (100, 'bar'),
+    ),
+)
+
+entry(
+    index = 2,
+    label = "H + CH3 <=> CH4",
+    degeneracy = 6.0,
+    kinetics = Arrhenius(
+        A = (2.94005e-11, 'm^3/(mol*s)'),
+        n = 5.135,
+        Ea = (33.0118, 'kJ/mol'),
+        T0 = (1, 'K')),
+    elementary_high_p = True,
```

Comment:
Why isn't it necessary to label most of the other reactions this way?

Reply:
It is, you are right. I wasn't consistent in defining `generate_high_p_limit_kinetics()`.
I added a check for `elementary_high_p` now.